src/signals-unix.c

```diff
@@ -780,6 +780,10 @@ static void *signal_listener(void *arg)
             else if (exit_on_sigint) {
                 critical = 1;
             }
+            // FIXME: Skip this if force
```

Comment:
What's the best way to implement this? It seems like `jl_check_force_sigint()` also updates the force counters, which I don't think we want?

Reply:
We're not doing this for now; we can add a more convenient "hard-interrupt" mechanism separately